### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by kasperski95**
- removed info about `development` branch

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
+- increasing test coverage
+- fixing small bugs or adding small features
 ## Documentation fixes
 If you spot a typo, broken link, a mistake in the documentation, feel free to [create a PR](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork).
 ## New features or fixes
 Check if your issue is listed on the [Issues](https://github.com/software-mansion/protostar/issues) page. If the issue exists and is assigned to somebody, it will be resolved in the near future. Otherwise:
 1. Create a new issue.
-1. Read [the development guide](https://github.com/software-mansion/protostar#development)
-1. Use `development` as a base branch.
+1. Read [the development guide](https://github.com/software-mansion/protostar#development).
```
Comment (MaksymilianDemitraszek) [inline CONTRIBUTING.md:12]: 1, 2, 3, 4 ...

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,20 +1,20 @@
 # Contributing Guide
+
+If you want to help develop this project, and you don't know where to start, consider:
```
Comment (MaksymilianDemitraszek) [inline CONTRIBUTING.md:3]: We could add a line that we welcome contributions. Also we can encourage asking questions on discord before starting a work

**Review by MaksymilianDemitraszek — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (kasperski95) [reply CONTRIBUTING.md:12]: I think markdown converter will automatically replace it with correct numeration.

Comment (MaksymilianDemitraszek) [reply CONTRIBUTING.md:12]: Okey

**Pull request merged**